[bashible] if internalNetworkCIDRs is not specified and 1 interface cannot define an ip for a new node

### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.56.9

### Expected Behavior

If there is only one interface on the node, you should put a single ip in the variable value.
or the error should describe the situation more clearly

### Actual Behavior

node is not added to the cluster if it is in a different l2 segment from the master.

### Steps To Reproduce

_No response_

### Additional Information

 https://t.me/deckhouse_ru/30232
 https://github.com/deckhouse/deckhouse/blob/d971287d8855fa61b7ecbc80ce669f40580622dd/candi/bashible/common-steps/node-group/000_discover_node_ip.sh.tpl#L88

### Logs

```shell
bash [ERROR] Failed to discover node_ip but its required for static cluster bootstrap or static cluster nodes
Failed to execute step /var/lib/bashible/bundle_steps/000_discover_node_ip.sh ... retry in 10 seconds.
```

--- a/candi/bashible/common-steps/all/000_discover_node_ip.sh.tpl
+++ b/candi/bashible/common-steps/all/000_discover_node_ip.sh.tpl
@@ -85,7 +85,7 @@ done
 
 {{- if or (eq .runType "ClusterBootstrap") (eq .nodeGroup.nodeType "Static") }}
 if [ -z "$(cat /var/lib/bashible/discovered-node-ip)" ] ; then
-  bb-log-error "Failed to discover node_ip but its required for static cluster bootstrap or static cluster nodes"
+  bb-log-error "Failed to discover node_ip that matches internalNetworkCIDRs."
   exit 1
 fi
 {{- end }}

